app/web/components/Button/Button.tsx

```diff
@@ -49,7 +49,7 @@ export type AppButtonProps<D extends ElementType = "button"> =
     classes?: Partial<ButtonClasses>; // Use the flexible ButtonClasses type here
   };
 
-function _Button<D extends ElementType = "button">(
```

Comment:
Eslint started getting mad about the underscore after upgrading `eslint-config-next`.